components/factories/AssessmentGrader/AssessmentGrader.tsx

```diff
@@ -24,8 +26,10 @@ const AssessmentGrader = ({
   sidebarMetadata,
   assessmentData,
   grader,
-  loadingStatus
+  loadingStatus,
 }: Props) => {
+  const router = useRouter();
+  console.log(submission);
```

Comment:
Hello, Alex. I think this should be removed :) Qapla!

Reply:
Oh yes, Jo. Thanks for the headsup!